Loop/Managers/RemoteDataServicesManager.swift

```diff
@@ -361,6 +361,21 @@ extension RemoteDataServicesManager {
 
 }
 
```

Comment:
Check our data services for one to validate our notification here. As a future enhancement, data services like Nightscout could provide some kind of identifier in the push payload. That would allow Loop to dispatch the notification to the corresponding service rather than sending it to all the services like this.